Python/bytecodes.c

```diff
@@ -1724,6 +1727,7 @@ dummy_func(
             STACK_SHRINK(shrink_stack);
             new_frame->localsplus[0] = owner;
             JUMPBY(INLINE_CACHE_ENTRIES_LOAD_ATTR);
+            frame->return_offset = 0;
```

Comment:
What are the rules about when this needs to be 0ed and when it doesn't?

Reply:
`return_offset` is only meaningful to the callee, so it needs to be set in any `CALL` (to a Python function) or `SEND` (to a coroutine or generator).
If there is no callee, then it is meaningless.

Reply:
I've added a comment in pycore_frame.h